recipes/pyspark/pysparkretailapp/data_saver.py

```diff
@@ -53,9 +53,11 @@ def save(self, config_properties, prediction):
 
         dataset_options = get_dataset_options(sparkContext)
 
-        scored_df.select(tenant_id, "_id", "eventType", "timestamp").write.format("com.adobe.platform.dataset") \
-            .option(dataset_options.orgId(), org_id) \
-            .option(dataset_options.serviceToken(), service_token) \
-            .option(dataset_options.userToken(), user_token) \
-            .option(dataset_options.serviceApiKey(), api_key) \
-            .save(scored_dataset_id)
+        scored_df.select(tenant_id, "_id", "eventType", "timestamp").write.format("com.adobe.platform.query") \
+        .option(dataset_options.userToken(), user_token) \
+        .option(dataset_options.serviceToken(), service_token) \
+        .option(dataset_options.imsOrg(), org_id) \
+        .option(dataset_options.apiKey(), api_key) \
+        .option(dataset_options.mode(), "interactive") \
```

Comment:
The committed code needs to be batch (even though we have some challenges with it right now.